sdk/template/Azure.Template/stress/Azure.Template.Stress.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
For the directory structure should we put it under "tests"? I guess if we do there will be some work to keep the projects independent because most tests put their project in the root of the tests folder. I'm Ok with using "<package>/stress", just wanted to mention it.

Reply:
I'm open to whatever you or the language team wants here.  I made "stress" a sibling of "tests" for exactly this reason, I didn't want to break any existing tests.